wpiunits/generate_units.py

```diff
@@ -46,6 +46,7 @@ def output(outPath, outfn, contents):
     "LinearAcceleration",
     "LinearMomentum",
     "LinearVelocity",
+    "LinearVelocityFeedForward",
```

Comment:
I'm not sold on this name. A linear velocity feedforward has three units: the input unit (e.g., velocity), the output unit (e.g., voltage, current), and the feedforward gain unit (output unit over input unit).

Reply:
How about FeedforwardVoltagePerLinearVelocity?

Reply:
VoltagePerDistancePerTime